benchmarks/benchmarker.py

```diff
@@ -0,0 +1,329 @@
+import numpy as np
+import git
+import json
+import os
+import argparse
+import sys
+import unittest
+import importlib
+
+
+# To add a new benchmark,
```

Comment:
This description of adding benchmarks should end up in the freud documentation, right below the Unit Tests in the developer guide. https://freud.readthedocs.io/en/stable/design.html#unit-tests